pkg/cluster/healthcheck.go

```diff
@@ -68,3 +68,8 @@ func (u *simpleUpgrader) HealthCheck(ctx context.Context, cs *api.OpenShiftManag
 	u.log.Info("checking admin console health")
 	return u.doHealthCheck(ctx, getHealthCheckHTTPClient(cs), "https://console."+cs.Properties.RouterProfiles[0].PublicSubdomain, 10*time.Second)
 }
+
+func (u *simpleUpgrader) WaitForHTTPStatusOk(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
```

Comment:
This name is confusing now. It is checking master healthz endpoint, but name is not define what we are checking :man_shrugging:

Reply:
Probably makes sense to update this or create a follow up (issue/PR) that does.